### PR TITLE
[HOPSWORKS-1379] Timers should be rescheduled when failed and handle exceptions

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/587

File: attributes/default.rb [hunk 1]
```diff
@@ -63,6 +63,8 @@
 default['hopsworks']['http_logs']['enabled']     = "true"
 default['hopsworks']['env_var_file']             = "#{node['hopsworks']['domains_dir']}/#{node['hopsworks']['domain_name']}_environment_variables"
 
+default['glassfish']['reschedule_failed_timer']     = "true"
```
Comment (tkakantousis) [inline attributes/default.rb]: @robzor92 nitpick, shouldn't it be plural? `timers`

Comment (robzor92) [reply attributes/default.rb]: I used the same naming scheme they do in the asadmin command, is it ok?

Comment (tkakantousis) [reply attributes/default.rb]: of course, it seems more logical to me to have it in plural but lets stick to what they do :)

**Pull request merged**